src/enums/authorizedActions.ts

```diff
@@ -3,29 +3,29 @@
  * Ultimately, this will be combined into the Authorization Roles enum.
  */
 export enum AuthorizedActions {
-  AMALGAMATION_FILING,
-  AML_OVERRIDES,
-  BLANK_CERTIFY_STATE,
-  BLANK_COMPLETING_PARTY,
-  CONTINUATION_IN_FILING,
-  COURT_ORDER_POA,
-  EDITABLE_CERTIFY_NAME,
-  EDITABLE_COMPLETING_PARTY,
-  FILE_AND_PAY,
-  FIRM_ADD_BUSINESS,
-  FIRM_DISSOLUTION_FILING,
-  FIRM_EDITABLE_EMAIL_ADDRESS,
-  FIRM_NO_MIN_START_DATE,
-  INCORPORATION_APPLICATION_FILING,
-  NO_COMPLETING_PARTY_MESSAGE_BOX,
-  NO_CONTACT_INFO,
-  REGISTRATION_FILING,
-  RESTORATION_REINSTATEMENT_FILING,
-  SAVE_DRAFT,
-  SBC_BREADCRUMBS,
-  STAFF_BREADCRUMBS,
-  STAFF_COMMENTS,
-  STAFF_PAYMENT,
-  THIRD_PARTY_CERTIFY_STMT,
-  VOLUNTARY_DISSOLUTION_FILING // non-firm voluntary dissolution
+  AMALGAMATION_FILING = 'AMALGAMATION_FILING',
+  AML_OVERRIDES = 'AML_OVERRIDES',
+  BLANK_CERTIFY_STATE = 'BLANK_CERTIFY_STATE',
+  BLANK_COMPLETING_PARTY = 'BLANK_COMPLETING_PARTY',
+  CONTINUATION_IN_FILING = 'CONTINUATION_IN_FILING',
+  COURT_ORDER_POA = 'COURT_ORDER_POA',
+  EDITABLE_CERTIFY_NAME = 'EDITABLE_CERTIFY_NAME',
+  EDITABLE_COMPLETING_PARTY = 'EDITABLE_COMPLETING_PARTY',
+  FILE_AND_PAY = 'FILE_AND_PAY',
+  FIRM_ADD_BUSINESS = 'FIRM_ADD_BUSINESS',
+  FIRM_DISSOLUTION_FILING = 'FIRM_DISSOLUTION_FILING',
+  FIRM_EDITABLE_EMAIL_ADDRESS = 'FIRM_EDITABLE_EMAIL_ADDRESS',
+  FIRM_NO_MIN_START_DATE = 'FIRM_NO_MIN_START_DATE',
+  INCORPORATION_APPLICATION_FILING = 'INCORPORATION_APPLICATION_FILING',
+  NO_COMPLETING_PARTY_MESSAGE_BOX = 'NO_COMPLETING_PARTY_MESSAGE_BOX',
+  NO_CONTACT_INFO = 'NO_CONTACT_INFO',
+  REGISTRATION_FILING = 'REGISTRATION_FILING',
+  RESTORATION_REINSTATEMENT_FILING = 'RESTORATION_REINSTATEMENT_FILING',
+  SAVE_DRAFT = 'SAVE_DRAFT',
+  SBC_BREADCRUMBS = 'SBC_BREADCRUMBS',
+  STAFF_BREADCRUMBS = 'STAFF_BREADCRUMBS',
+  STAFF_COMMENTS = 'STAFF_COMMENTS',
+  STAFF_PAYMENT = 'STAFF_PAYMENT',
+  THIRD_PARTY_CERTIFY_STMT = 'THIRD_PARTY_CERTIFY_STMT',
+  VOLUNTARY_DISSOLUTION_FILING = 'VOLUNTARY_DISSOLUTION_FILING' // non-firm voluntary dissolution
```

Comment:
We get the strings, not enum values (eg, 14) from the API, so this enum has to be string values.